src/utils/formula-collection.ts

```diff
@@ -0,0 +1,490 @@
+import { Formula } from "@/schema/formula";
+
+// This should all be gotten from the EGRID and AVERT databases
+export const AVERT_AND_EGRID = {
+  // energyType value map is as follows:
+  // Consumed is 0, Onshore Wind is 1, Offshore Wind is 2, Utility PV is 3, Distributed PV is 4, Portfolio EE is 5, Uniform EE is 6
+  // This is E2 on the spreadsheet
+  energyType: 2,
+  regional: 1, // 1 for regional, 0 for national
+
+  kWInstalledCapacity: 5882000,
+  capacityFactor: 0.51,
+
+  nationalEmissionRateFromElectricityConsumed: 823.15,
+  regionalCaliforniaEmissionRateFromElectricityConsumed: 455.94,
+
+  regionalOnshoreWindAvoidedCO2: 946.5,
+  nationalOnshoreWindAvoidedCO2: 1308,
+
+  regionalOffshoreWindAvoidedCO2: 948.1,
+  nationalOffshoreWindAvoidedCO2: 1228,
+
+  regionalUtilityPVAvoidedCO2: 949.4,
+  nationalUtilityPVAvoidedCO2: 1347,
+
+  regionalDistributedPVAvoidedCO2: 1039.6,
+  nationalDistributedPVAvoidedCO2: 1449,
+
+  regionalPortfolioEEAvoidedCO2: 1058,
+  nationalPortfolioEEAvoidedCO2: 1455,
+
+  regionalUniformEEAvoidedCO2: 1037,
+  nationalUniformEEAvoidedCO2: 1429,
+};
+
+/*
+    Impact Calculator Equation 1: Conversion between Electricity Consumed and Reduced
+ */
+
+// This is representative of the user input calculation, which is required for conversionBetweenElectricityConsumedAndReduced
+// Figured it would be useful to have in formula form for evaluation of user inputs
+export const annualPowerGeneration: Formula = {
+  id: "annualPowerGeneration",
+  name: "Annual Power Generation",
+  explanation: "Takes user input of kW installed capacity and capacity factor, and calculates annual power generation",
+  assumptions: [""],
+  sources: ["User input"],
+  expression: "kWInstalledCapacity * capacityFactor * 8760",
+  unit: "",
+  setupScope: () => {},
+  dependencies: ["kWInstalledCapacity", "capacityFactor"],
+};
+
+export const CO2PerkWhConsumed: Formula = {
+  id: "CO2PerkWhConsumed",
+  name: "CO2 per kilowatt-hour consumed",
+  explanation: "",
```

Comment:
Would it be possible to add a little explanation to describe what these intermediate formulas do?